projects/lights_control/test/test_lights_can.c

```diff
@@ -0,0 +1,147 @@
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_pack.h"
+#include "event_queue.h"
+#include "interrupt.h"
+#include "status.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#include "exported_enums.h"
+#include "lights_can.h"
+#include "lights_events.h"
+
+// Using SYSTEM_CAN_DEVICE_LIGHTS_REAR to be able to test sync message
+#define TEST_LIGHTS_CAN_DEVICE_ID SYSTEM_CAN_DEVICE_LIGHTS_REAR
+
+#define TEST_LIGHTS_CAN_RX_ADDR \
+  { .port = 0, .pin = 0 }
```

Comment:
You need to use actual pins to run this on stm32.

Reply:
thanks for heads up, I'll change it.